site/book/01-getting-started/02-quickstart.md

```diff
@@ -44,21 +44,34 @@ $ vim deployment.yaml
 Often, you want to automatically mutate and/or validate resources in a package.
 `kpt fn` commands enable you to execute programs called _kpt functions_.
 
-For example, you can automatically set a label with key `env` on all the resources in the package:
+For example, you can automatically search and replace all the occurrences of `app` name on resources
+in the package using [path expressions]:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/set-label:v0.1 -- env=dev
+$ kpt fn eval --image gcr.io/kpt-fn/search-replace:v0.1 -- 'by-path=spec.**.app' 'put-value=my-nginx'
 ```
 
 `eval` command can be used for one-time _imperative_ operations. For operations that need to be
 performed repeatedly, there is a _declarative_ way to define a pipeline of functions as part of the
-package (in the `Kptfile`). This pipeline is executed using the `render` command:
+package (in the `Kptfile`). For example, you might want organize resources in the package using a label `env`.
```

Comment:
For example, you may want to add a label to all resources in a package ("organize" has a different connotation)

Reply:
Borrowed the word from labels description(https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/). Updated it anyways as it is simpler.